protected/humhub/modules/user/models/forms/AccountSettings.php

```diff
@@ -35,7 +36,7 @@ public function rules()
             [['show_introduction_tour'], 'boolean'],
             [['timeZone'], 'in', 'range' => \DateTimeZone::listIdentifiers()],
             ['language', 'in', 'range' => array_keys(Yii::$app->i18n->getAllowedLanguages())],
-            ['visibility', 'in', 'range' => [1, 2]],
+            ['visibility', 'in', 'range' => array_keys(User::getVisibilityOptions())],
```

Comment:
Could a user then mark themselves as Hidden?

Reply:
@luke- Yes, it was possible to set this by hack. Fixed in commit https://github.com/humhub/humhub/pull/5607/commits/d78d1428bad0b0d01b7073fe2a8e577c6028f6f1.